.github/workflows/update.yaml

```
@@ -21,14 +21,14 @@ jobs:
           fetch-depth: 0
 
       - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v3
```

nodejs 16 runner deprecated로 v5버전 사용해야합니다
```suggestion
        uses: actions/setup-python@v5
```